integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -12,8 +12,8 @@
           enctype="multipart/form-data"
           data-unsaved-warning>
         {% csrf_token %}
-        <div class="w-full flex flex-wrap justify-between mb-4">
-            <h1 class="heading overflow-hidden text-ellipsis">
+        <div class="flex flex-wrap justify-between w-full mb-4">
```

Comment:
Did you use a tool to sort the classes alphabetical? If so, I'd like to include this into our code style tools :grin:

Reply:
Yes, I did. I use the [Headwind extension](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind) for VS Code :)